src/commands/npm/release/validate.ts

```diff
@@ -41,8 +43,15 @@ export default class Validate extends SfdxCommand {
       });
       responses.push(response);
     }
-    const shouldRelease = responses.some((resp) => !!resp.shouldRelease);
+    const majorBump = responses.some((resp) => !!resp.isMajorBump);
```

Comment:
the error condition should be
1. commits have some breaking change ✅ 
2. the pjson major version was not manually bumped ❓  

Maybe I don't understand when this code runs?  Is it only when the pjson wasn't manually modified?  Otherwise, how would you not hit the error when you are doing a major bump intentionally?

Reply:
@mshanemc The manual bump logic is here: https://github.com/salesforcecli/plugin-release-management/blob/main/src/inspectCommits.ts#L66

If the version has been manually bumped, then `shouldRelease` always is set to `true` and `isMajorBump` is left unset. So in that case, it'll never throw the error